dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/EventRequestToSearchParamsMapperTest.java

```diff
@@ -539,10 +534,9 @@ void testFilterAttributesWhenSameTEAHasMultipleFilters()
         assertNotNull( exception.getMessage() );
```

Comment:
We can get rid of this assertNotNull, right?
It should be included in the new assertion

Reply:
good catch, will remove it right away 😄

Reply:
oh, sorry just saw that there are other assertions in the assertAll that would get a NPE if I remove it. I have a follow up where I do remove it as I also add an assertContains. So it will be removed, just not in this PR 😋

Reply:
👍